integrations/docker/images/chip-build-cirque/requirements_nogrpc.txt

```diff
@@ -1,6 +1,6 @@
 cmd2
 docker >= 4.1.0
-flask == 2.2.2
+flask == 2.2.5
```

Comment:
Wait, this would basically revert https://github.com/project-chip/connectedhomeip/pull/26306, no?  @yunhanw-google which version of flask was not OK here?